app/views/late_reservation_mailer/forgot_to_buy_gift_email.html.erb

```diff
@@ -0,0 +1,14 @@
+  body
+    h1Your reservation has not been ordered yet!/h1
```

Comment:
This is a syntax error.

You should have a space after each tag name in Slim - and you don't need to close tags (`/h1` will render as the text `/h1` in the template)